GitWorkflow/GitWorkflow/Level.cpp

```diff
@@ -1,10 +1,17 @@
 #include "Level.hpp"
+#include <assert.h>
 
 Level::Level()
 	: m_player(), m_endPoint(), m_objects()
 {
 }
 
+Player& Level::getPlayer() const
+{
+	assert(m_player);
+	return *m_player;
+}
+
```

Comment:
Personally, I'm not a fan of asserts in product code.
I would prefer std::optionals or pointers in this case.